### PR TITLE
 Add Travis-CI integration to project, but with limited scope

### PR DESCRIPTION
**Pull request by ottok**

### CODE REVIEW

**Comment by ottok** [comment]
Please review and merge #93 and #95 first.

**Comment by ericzolf** [comment]
#93 and #95 have been merged, aren't you also dependent on the Dockerfile?

**Comment by ottok** [comment]
Running Tox on Travis-CI does not depend on the Dockerfile, but the Dockerfile will play an important role in futher CI developments and also general developer/debugging reproducibility.

**Comment by ericzolf** [comment]
I assume that would be bad to merge this PR until it's successful (which probably means a complete flake8 clean-up).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - libacl1-dev
+    - librsync-dev
+    - rdiff
+
+install:
+  - pip install -U pip tox virtualenv
+
+matrix:
+  include:
+    - python: 3.6
```
Comment (ericzolf) [inline .travis.yml]: Why Python 3.6? It seemed to me our reference implementation is 3.7.

Comment (ottok) [reply .travis.yml]: I had 3.6 because that was what Ubuntu 18.04 ships, but sure, let's keep 3.7 as our main target and use it for flake8 as well.

**Comment by ottok** [comment]
Merging is still blocked and anyway this was now obsoleted by https://github.com/rdiff-backup/rdiff-backup/pull/116, so closing.

**Pull request closed without merging**